From 9e9b23b614d9b67b9c7a4f9d5c2677249062df0f Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Tue, 19 Dec 2017 21:00:28 +0800
Subject: [PATCH 3117/5242] MLK-17242-2 arm64: dts: fsl-imx8qxp-mek: remove
 properties for power sink

commit  397e4a543493b24c4f945a5cde38c68a112db0e8 from
https://source.codeaurora.org/external/imx/linux-imx.git

As imx8qxp MEK typec port only can support power source on power role,
remove those properties for power sink after we add fixed power sink
settings.

Tested-by: Peter Chen <peter.chen@nxp.com>
Acked-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 06e4f26..32d514d 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -653,10 +653,6 @@
 		ss-sel-gpios = <&gpio5 9 GPIO_ACTIVE_LOW>;
 		reset-gpios = <&pca9557_a 7 GPIO_ACTIVE_HIGH>;
 		src-pdos = <0x380190c8 0x3803c0c8>;
-		snk-pdos = <0x380190c8>;
-		max-snk-mv = <9000>;
-		max-snk-ma = <1000>;
-		op-snk-mw = <9000>;
 		port-type = "drp";
 		sink-disable;
 		default-role = "source";
-- 
1.7.9.5

